client/__tests__/e2e/e2e-jest-config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
we are configuring the unit test jest inside package.json, but doing it as stand-alone files for puppeteer.     Should they both use the same method?   (I don't know the answer - just asking why we would do it differently)

Reply:
The jest-puppeteer package has configuration that break the presets for the unit tests. My understanding is that most future tests will be unit tests and so I feel like that should be the default configuration. I wish there was an easy way to set up alternate configurations for tests in different folders but my googling didn't bring up anything. Totally open to suggestions

Reply:
I wasn't proposing an identical config.  Rather that we have the structure of the config be the same (ie, all in package.json, or all in stand-alone files).   

That said, this is a super tiny issue, so if not easy, skip it.

Reply:
I was thinking of it as the unit test config should be the default (thus in package.json) and the smoke test should be the exception (thus in a separate file). IMO future exceptions should have separate files as well.

Reply:
SGTM